pkg/exemplars/multitsdb.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package exemplars
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	"github.com/thanos-io/thanos/pkg/exemplars/exemplarspb"
+)
+
+// MultiTSDB implements exemplarspb.ExemplarsServer that allows to fetch exemplars a MultiTSDB instance.
+type MultiTSDB struct {
+	tsdbExemplarsServers func() map[string]exemplarspb.ExemplarsServer
+}
+
+// NewMultiTSDB creates new exemplars.MultiTSDB.
+func NewMultiTSDB(tsdbExemplarsServers func() map[string]exemplarspb.ExemplarsServer) *MultiTSDB {
+	return &MultiTSDB{
+		tsdbExemplarsServers: tsdbExemplarsServers,
+	}
+}
+
+// Exemplars returns all specified exemplars from a MultiTSDB instance.
+func (m *MultiTSDB) Exemplars(r *exemplarspb.ExemplarsRequest, s exemplarspb.Exemplars_ExemplarsServer) error {
+	fmt.Println("meer paas request aayi", m.tsdbExemplarsServers())
```

Comment:
:D

Reply:
Yep, I know it's here haha. I'll fix this and push XD